light-zk.js/src/transaction/transaction.ts

```diff
@@ -643,7 +638,7 @@ export class Transaction {
 
   // TODO: evaluate whether confirm options should be here or in transaction parameters
   async sendTransaction(
-    confirmOptions: ConfirmOptions = ConfirmOptions.finalized,
+    _confirmOptions: ConfirmOptions = ConfirmOptions.finalized,
```

Comment:
this can be removed
including the todo

Reply:
if I remove it, cascading I have to deal with this:

```
  async sendAndConfirmTransaction(
    confirmOptions: ConfirmOptions = ConfirmOptions.spendable,
  ): Promise<
    RelayerSendTransactionsResponse | SendVersionedTransactionsResult
  > {
    return await this.sendTransaction(confirmOptions);
  }
```

this calls the ```this.sendTransaction``` and passes to it a ```confirmOption``` param

Reply:
sure you can remove confirm options in both places

Reply:
removed ✅